controllers/grid/pubIds/PubIdExportIssuesListGridHandler.inc.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/**
+ * @file controllers/grid/pubIds/PubIdExportIssuesListGridHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class PubIdExportIssuesListGridHandler
+ * @ingroup controllers_grid_pubIds
+ *
+ * @brief Handle exportable submissions with pub ids list grid requests.
+ */
+
+import('lib.pkp.classes.controllers.grid.GridHandler');
+
+import('controllers.grid.pubIds.PubIdExportIssuesListGridCellProvider');
+
+class PubIdExportIssuesListGridHandler extends GridHandler {
+	/** @var boolean true if the current user has a managerial role */
+	var $_isManager;
+
+	/** @var ImportExportPlugin */
+	var $_plugin;
+
+	/**
+	 * Constructor
+	 */
+	function PubIdExportIssuesListGridHandler() {
+		parent::GridHandler();
+		$this->addRoleAssignment(
+			array(ROLE_ID_MANAGER),
+			array('fetchGrid', 'fetchRow')
+		);
+	}
+
+	//
+	// Implement template methods from PKPHandler
+	//
+	/**
+	 * @copydoc PKPHandler::authorize()
+	 */
+	function authorize($request, &$args, $roleAssignments) {
+		import('lib.pkp.classes.security.authorization.ContextAccessPolicy');
+		$this->addPolicy(new ContextAccessPolicy($request, $roleAssignments));
+		return parent::authorize($request, $args, $roleAssignments);
+	}
+
+	/**
+	 * @copydoc PKPHandler::initialize()
+	 */
+	function initialize($request) {
+		parent::initialize($request);
+
+		// Basic grid configuration.
+		$this->setTitle('plugins.importexport.common.export.issues');
+
+		// Load submission-specific translations.
+		AppLocale::requireComponents(
+			LOCALE_COMPONENT_APP_COMMON,
+			LOCALE_COMPONENT_APP_SUBMISSION,
+			LOCALE_COMPONENT_PKP_SUBMISSION,
+			LOCALE_COMPONENT_APP_EDITOR
+		);
+
+		// Fetch the authorized roles and determine if the user is a manager.
+		$authorizedRoles = $this->getAuthorizedContextObject(ASSOC_TYPE_USER_ROLES);
+		$this->_isManager = in_array(ROLE_ID_MANAGER, $authorizedRoles);
+
+		$pluginCategory = $request->getUserVar('category');
+		$pluginPathName = $request->getUserVar('plugin');
+		$this->_plugin = PluginRegistry::loadPlugin($pluginCategory, $pluginPathName);
```

Comment:
:-)
Hmmm.. Because it is integration with load_url_in_div, maybe to implement a new CrossrefSubmissionListGridHandler that will extend this one and set the plugin variable?